src/lib/intl-polyfill.js

```diff
@@ -19,293 +21,458 @@ import {shouldPolyfill as shouldPolyfillPluralRules} from '@formatjs/intl-plural
+        await import('@formatjs/intl-numberformat/locale-data/he');
+        await import('@formatjs/intl-datetimeformat/locale-data/he');
         break;
     case 'hr':
-        await import('@formatjs/intl-relativetimeformat/locale-data/hr');
         await import('@formatjs/intl-pluralrules/locale-data/hr');
+        await import('@formatjs/intl-relativetimeformat/locale-data/hr');
+        await import('@formatjs/intl-numberformat/locale-data/hr');
+        await import('@formatjs/intl-datetimeformat/locale-data/hr');
         break;
     case 'xh':
-        await import('@formatjs/intl-relativetimeformat/locale-data/xh');
         await import('@formatjs/intl-pluralrules/locale-data/xh');
+        await import('@formatjs/intl-relativetimeformat/locale-data/xh');
+        await import('@formatjs/intl-numberformat/locale-data/xh');
+        await import('@formatjs/intl-datetimeformat/locale-data/xh');
         break;
     case 'zu':
-        await import('@formatjs/intl-relativetimeformat/locale-data/zu');
         await import('@formatjs/intl-pluralrules/locale-data/zu');
+        await import('@formatjs/intl-relativetimeformat/locale-data/zu');
+        await import('@formatjs/intl-numberformat/locale-data/zu');
+        await import('@formatjs/intl-datetimeformat/locale-data/zu');
         break;
     case 'is':
-        await import('@formatjs/intl-relativetimeformat/locale-data/is');
         await import('@formatjs/intl-pluralrules/locale-data/is');
+        await import('@formatjs/intl-relativetimeformat/locale-data/is');
+        await import('@formatjs/intl-numberformat/locale-data/is');
+        await import('@formatjs/intl-datetimeformat/locale-data/is');
         break;
     case 'it':
-        await import('@formatjs/intl-relativetimeformat/locale-data/it');
         await import('@formatjs/intl-pluralrules/locale-data/it');
+        await import('@formatjs/intl-relativetimeformat/locale-data/it');
+        await import('@formatjs/intl-numberformat/locale-data/it');
+        await import('@formatjs/intl-datetimeformat/locale-data/it');
         break;
     case 'ka':
-        await import('@formatjs/intl-relativetimeformat/locale-data/ka');
         await import('@formatjs/intl-pluralrules/locale-data/ka');
+        await import('@formatjs/intl-relativetimeformat/locale-data/ka');
+        await import('@formatjs/intl-numberformat/locale-data/ka');
+        await import('@formatjs/intl-datetimeformat/locale-data/ka');
         break;
     case 'kk':
-        await import('@formatjs/intl-relativetimeformat/locale-data/kk');
         await import('@formatjs/intl-pluralrules/locale-data/kk');
+        await import('@formatjs/intl-relativetimeformat/locale-data/kk');
+        await import('@formatjs/intl-numberformat/locale-data/kk');
+        await import('@formatjs/intl-datetimeformat/locale-data/kk');
+        break;
+    case 'qu':
+        await import('@formatjs/intl-pluralrules/locale-data/en');
```

Comment:
is there a reason the other imports here use `qu` but this one uses `en`?

Reply:
Good question. Despite using the latest version of `intl-pluralrules`, `intl-relativetimeformat`, `intl-numberformat`, and `intl-datetimeformat`, they don't support the same set of locales. This is a case of `qu` missing from `intl-pluralrules` but not the others. When `react-intl` can't find a locale, it looks like it falls back to `en`, so let's import `en` just to be safe.

I pretty much copied this behavior from our previous code. I hope I'm interpreting the intent correctly. The line you pointed out for `mi` existed before this PR. 

I didn't want to do this all by hand. I wrote a script to do it. Posting the important parts here in case it helps someone in the future.

```js
const {supportedLocales: pluralRulesSupportedLocales} = require('@formatjs/intl-pluralrules/supported-locales.generated');
const {supportedLocales: relativeTimeFormatSupportedLocales} = require('@formatjs/intl-relativetimeformat/supported-locales.generated');
const {supportedLocales: numberFormatLocales} = require('@formatjs/intl-numberformat/supported-locales.generated');
const {supportedLocales: dateTimeFormatLocales} = require('@formatjs/intl-datetimeformat/supported-locales.generated');

const languages = {
...
};

for (const locale of Object.keys(languages).map(l => l.toLowerCase().split('-')[0])) {
    console.log(`    case '${locale}':`);
    console.log(`        await import('@formatjs/intl-pluralrules/locale-data/${translateLocale(locale, pluralRulesSupportedLocales)}');`);
    console.log(`        await import('@formatjs/intl-relativetimeformat/locale-data/${translateLocale(locale, relativeTimeFormatSupportedLocales)}');`);
    console.log(`        await import('@formatjs/intl-numberformat/locale-data/${translateLocale(locale, numberFormatLocales)}');`);
    console.log(`        await import('@formatjs/intl-datetimeformat/locale-data/${translateLocale(locale, dateTimeFormatLocales)}');`);
    console.log('        break;');
}

function translateLocale(locale, supportedLocales) {
    return supportedLocales.includes(locale) ? locale : 'en';
}
```

Reply:
makes sense! just wanted to make sure I understood what was going on there. thanks!